src/detect-uricontent.c

```diff
@@ -1743,36 +934,25 @@ static int DetectUricontentIsdataatParseTest(void)
 
 static void DetectUricontentRegisterTests(void)
 {
-    UtRegisterTest("HTTPUriTest01", HTTPUriTest01);
-    UtRegisterTest("HTTPUriTest02", HTTPUriTest02);
-    UtRegisterTest("HTTPUriTest03", HTTPUriTest03);
-    UtRegisterTest("HTTPUriTest04", HTTPUriTest04);
-
-    UtRegisterTest("DetectUriSigTest01", DetectUriSigTest01);
+    UtRegisterTest("DetectUriSigTest04 - Modifiers", DetectUriSigTest01);
```

Comment:
I think with all the deletion and renaming, this one got messed up a bit? If `DetectUriSigTest01` is meant to stay, then let's please keep its original name, too.